index.html

```diff
@@ -11,6 +11,17 @@
 
     <header class="app-header">
         <a href="#" class="logo">Narson</a>
+
+        
+        <div id="myNav" class="overlay">
+            <a href="javascript:void(0)" class="closebtn">&times;</a>
+            <div class="overlay-content">
+                <a id="portfolio" href="#work">Portfolio</a>
+                <a id="about" href="#about">About</a>
+                <a id="contact" href="#">Contact</a>
+            </div>
+        </div>
+
         <nav class="ambony">
             <a href="#"><img src="media/Union.svg" alt="humburger" class="humburger"></a>
```

Comment:
Your design
![image](https://github.com/rbd3/Narson-Portfolio/assets/78251927/fbb731f3-80f6-4dd3-90b9-a47c69b6d877)
 
Figma
![image](https://github.com/rbd3/Narson-Portfolio/assets/78251927/08399b9d-77ca-4f9a-9c63-c3568ac0fd7e)


- [ ] Please let's consider reducing the size of the hamburger menu `x` to match the design as possible. 🤗